cmd/rookflex/cmd/mount.go

```diff
@@ -244,6 +244,10 @@ func mountCephFS(client *rpc.Client, opts *flexvolume.AttachOptions) error {
 	)
 	if err != nil {
 		log(client, err.Error(), true)
+		log(client, "NOTE: Rook Flexvolume plugin uses 'mds_namespace' to refer to the ceph filesystem. "+
+			"This option is only available on Kernel v4.7 and newer. "+
```

Comment:
I would use "higher" instead of "newer". I don't really know why, but it just sounds better to me for version numbers.
@kokhang What do you think?

Reply:
sure :)